source/common/network/utility.h

```diff
@@ -91,6 +91,20 @@ class Utility {
   static bool isLoopbackAddress(const Address::Instance& address);
 
   /**
+   * @return a address that represents the IPv4 wildcard address (i.e. "0.0.0.0"). Used
+   * during binding to indicate that incoming connections to any local IPv4 address
+   * are to be received.
+   */
+  static Address::InstancePtr getIpv4AnyAddress();
```

Comment:
This seem mutable, which could lead to badness when shared. Can it be made const somehow?